internal/mysql/gtids/wrapper.go

```diff
@@ -6,10 +6,10 @@ import (
 
 type GTIDSet = mysql.GTIDSet
 
-func ParseGtidSet(gtidset string) mysql.GTIDSet {
-	parsed, err := mysql.ParseGTIDSet(mysql.MySQLFlavor, gtidset)
+func ParseGtidSet(gtidset string) *mysql.MysqlGTIDSet {
```

Comment:
Do we really need here specific type instead of generic?
I guess it will be better to check type inside isSplitBrained